native/dgeqrf.go

```diff
@@ -14,7 +14,7 @@ import (
 // parameters at entry and exit.
 //
 // work is temporary storage, and lwork specifies the usable memory length.
-// At minimum, lwork >= m and this function will panic otherwise.
+// At minimum, lwork >= max(1, m) and this function will panic otherwise.
 // Dgeqrf is a blocked QR factorization, but the block size is limited
```

Comment:
Done. This required a moderate rewording since that would have been incorrect as well.